src/pcapng.h

```diff
@@ -58,7 +63,7 @@ struct PCAPNG_BLOCK_OPTION_EPB_FLAGS {
     unsigned short Length;        // 4
     unsigned long  Value;
 };
-struct PCAPNG_BLOCK_OPTION_COMMENT {
+struct PCAPNG_BLOCK_OPTION_STRING {
```

Comment:
Don't rename this.

Reply:
I reused this. It's not a comment anymore.

Reply:
So the comment next to the Code field (// PCAPNG_OPTIONCODE_COMMENT) is now incorrect?

Pcapng is a simple file format and we don't need to reuse structs in tricky ways to minimize the number of lines of code. If you're implementing a new struct from the spec, please just make a new struct, even if it's identical to an existing one.

Reply:
ok sgtm

Reply:
Discussed offline; we'll use this generic struct with an appropriately generic name and fix the comment in the Code field.